src/gigs-board/components/layout/FeaturedCommunity.jsx

```diff
@@ -0,0 +1,71 @@
+/* INCLUDE: "common.jsx" */
```

Comment:
I would create a folder for communities in the `components` folder, as there are quite a few components related to them.